File: 4051-remove-zeros-in-decimal-representation/4051-remove-zeros-in-decimal-representation.cs
public class Solution {
    public long RemoveZeros(long n) {
        StringBuilder str=new StringBuilder();
        str.Append(n);
        int i=0;
        int len=str.Length;
        while(i<len){
            if(str[i]=='0'){
                str.Remove(i,1);
                len--;
            }
            else{
                i++;
            }
        }
        long ans=long.Parse(str.ToString());
        return ans;
    }
}

File: 4051-remove-zeros-in-decimal-representation/4051-remove-zeros-in-decimal-representation.java
class Solution {
    public long removeZeros(long n) {
        StringBuilder str=new StringBuilder();
        str.append(n);
        int i=0;
        int n1=str.length();
        while(i<n1){
            if(str.charAt(i)=='0'){
                str.deleteCharAt(i);
                n1--;
            }
            else{
            i++;}
    
        }
        long ans = Long.parseLong(str.toString());;  
        return ans;
    }
}